runtime-modules/council/src/tests.rs

```diff
@@ -1188,19 +1197,16 @@ fn council_missed_rewards_are_paid_later() {
         MockUtils::increase_block_number(<Runtime as Trait>::ElectedMemberRewardPeriod::get());
 
         let last_payment_block = council_settings.cycle_duration
-            + (reward_period
-                - (council_settings.idle_stage_duration
-                    % <Runtime as Trait>::ElectedMemberRewardPeriod::get()))
-            - 1; // -1 because current block is not finalized yet
+            - (council_settings.idle_stage_duration
+                % <Runtime as Trait>::ElectedMemberRewardPeriod::get());
```

Comment:
I believe it was an artifact because the `run_full_cycle` actually ran 4 blocks more than a full cycle.

So with the current params(idle period 27 blocks and reward period 10) the last time a reward is paid is in the 20th block since the council was elected(the idle period started) but if we run the period for 4 extra blocks(until the 31st block after the idle period started) the last time a reward will be paid is the 30th block since the idle period started.